test/unit/senderSpec.js

```diff
@@ -73,25 +81,6 @@ describe('UNIT Sender', function () {
       }, 10);
     });
 
-    it('should not override internal request params if passed into constructor (except timeout/uri)', function (done) {
```

Comment:
Unfortunately, we will need this test to pass, it won't be possible to merge the PR with this test deleted. Please modify the test accordingly so it passes.

Reply:
@pmb-cl

Reply:
@eladnava thanks for the feedback. I put the test back.

I originally removed the test because I didn't think it was needed since we are using `.post()` now and it would post regardless of the method passed in.